src/components/shared/models/mentorship.ts

```diff
@@ -21,6 +21,8 @@ export class Mentorship {
   public User?: IUser;
   public Status?: string;
   public UserMessage?: string;
+  public createdAt?: string;
+  public updatedAt?: string;
```

Comment:
Change the casing here to follow convention

```suggestion
  public CreatedAt?: string;
  public UpdatedAt?: string;
```

Reply:
Questions, why did you include UpdateAt? This isn't currently part of the response.